code/modules/clothing/suits/utility.dm

```diff
@@ -107,7 +107,8 @@
 	name = "radiation hood"
 	icon_state = "rad"
 	desc = "A hood with radiation protective properties. The label reads, 'Made with lead. Please do not consume insulation.'"
-	flags = BLOCKHAIR|THICKMATERIAL
+	flags = THICKMATERIAL
+	flags_inv = HIDEMASK|HIDEEARS|HIDEFACE|HIDEEYES|HIDEHAIR|HIDEFACIALHAIR

```

Comment:
Rad, bio, and bomb hoods all have a slightly transparent faceplate, so they shouldn't hide masks or glasses